src/ui/Windows/res/PasswordSafe3.rc2

```diff
@@ -261,7 +261,9 @@ BEGIN
   IDS_PROVISMUTUALLYEXCL  "Sorry, 'pronounceable' and 'easy-to-read' are not supported together"
   IDS_DEFAULTPWLENGTHTOOSMALL "Password length is less than sum of 'at least' constraints"
   IDS_CANTSPECIFYMINNUMBER "Sorry, can't specify a minimum number of characters with 'pronounceable' option. Specified values ignored."
-  IDS_OWNSYMBOLSMISSING    "If you specify using your own symbol list, you cannot leave it empty. Unselecting use of symbols."
+  IDS_OWNSYMBOLSMISSING   "If you specify using your own symbol list, you cannot leave it empty. Unselecting use of symbols."
+  IDS_ADDEDITERR_INVALID_TOTP_KEY "Invalid Password Safe Authenticator configuration. %s (%d)"
```

Comment:
For consistency, should be "Invalid Authenticator Secret Key: %s"
For consistency, I suggest removing the "(%d)", since it's not used elsewhere in PasswordSafe.